pkg/internal/enable-containers.ps1

```diff
@@ -0,0 +1,8 @@
+$requiredWindowsFeature = "Containers"
```

Comment:
This is specific to `Containers`, so I suggest naming this as:
```
$ContainerFeatureName = "Containers"
```

On the other hand, you can make this extensible and have a collection with a list of the required features. (`requiredWindowsFeatures`).